components/fxa-client/src/login_sm.rs

```diff
@@ -1,294 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
I deleted the entire login state-machine, based on https://github.com/mozilla/application-services/pull/1005#pullrequestreview-228052263